src/main/java/io/vlingo/actors/Stage.java

```diff
@@ -739,6 +723,38 @@ private Definition redefinitionWithMailboxName(final Definition definition, fina
     return redefinition;
   }
 
+  <T> Actor rawLookupOrStart(Definition definition, Address address) {
+    Actor actor = directory.actorOf(address);
+    if (actor != null) {
+      return actor;
+    }
+    try {
+      return createRawActor(definition, definition.parentOr(world.defaultParent()), address, null, definition.supervisor(), world.defaultLogger());
```

Comment:
@VaughnVernon this one here, I'm not 100% sure if it is the correct way to use `createRawActor`.